### DISCUSSION TITLE
Bevy iOS app development

### DISCUSSION THREAD

**Discussion by nikolajevs86** [bevyengine/bevy#8694] [Q&A] [answered]
Hi! Is there any tutorial how to build iOS application with Bevy? I see that there is an example in bevy but there is no description how to run it

**Comment by mockersf** [comment, marked as answer]
There are rough explanations here: https://github.com/bevyengine/bevy/tree/latest/examples#ios

The [xcode project file](https://github.com/bevyengine/bevy/tree/latest/examples/mobile/bevy_mobile_example.xcodeproj) has a few custom settings enabled and can be driven either through the UI or this [makefile](https://github.com/bevyengine/bevy/blob/latest/examples/mobile/Makefile).

**Reply by nikolajevs86** [reply]
Thanks, also found this tutorial which helped me to run simple bevy app on ios sim 👍

**Comment by nikolajevs86** [comment]
Sorry for off topic but how I am able to store something on the iOS in some text file? If you are aware of course...
